internal/js/console.go

```diff
@@ -98,20 +100,87 @@ func (c console) valueString(v sobek.Value) string {
 		}
 	}
 
-	if sobekObj, isObj := v.(*sobek.Object); isObj {
-		if sobekObj.ClassName() == "Error" {
+	var (
+		mv any // either marshal from Sobek or Go
+		ok bool
+	)
+	obj, isObj := v.(*sobek.Object)
+	if isObj {
```

Comment:
I personally prefer to have this on one line when possible but no strong opinion: 
```suggestion
	if obj, isObj := v.(*sobek.Object); isObj {
```